common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -199,34 +202,32 @@ def _delete_student_state_for_problem(self):
         deleting the student user's state for the problem.
         """
         with self._logged_in_session(staff=True):
-            self.courseware_page.visit()
```

Comment:
**Note to reviewers:** After verifying with @sanfordstudent (as the original author of this test), I changed this test to call the Delete State functionality through the instructor dashboard instead of the through StaffDebug.  I did this so there was some acceptance test coverage of that UI location.